src/test/kotlin/no/nav/amt/arena/acl/processors/DeltakerStatusConverterTest.kt

```diff
@@ -275,8 +275,55 @@ class DeltakerStatusConverterTest : StringSpec({
 			gjennomforingStatus
 		).navn shouldBe PABEGYNT_REGISTRERING
 	}
-	"status - JATAKK - returnerer PABEGYNT" {
-		convert(TiltakDeltaker.Status.JATAKK, now, null, null, null, gjennomforingStatus).navn shouldBe PABEGYNT_REGISTRERING
+	"status - JATAKK - returnerer VENTER_PÅ_OPPSTART" {
+		convert(
+			TiltakDeltaker.Status.JATAKK,
+			now,
+			null,
+			null,
+			null,
+			gjennomforingStatus
+		).navn shouldBe VENTER_PA_OPPSTART
+	}
+	"status - JATAKK og har startdato i fortid - returnerer GJENNOMFORES" {
+		convert(
+			TiltakDeltaker.Status.JATAKK,
+			now,
+			yesterday,
+			null,
+			null,
+			gjennomforingStatus
+		).navn shouldBe DELTAR
+	}
+	"status - JATAKK og har startdato i fremtid - returnerer VENTER_PÅ_OPPSTART" {
+		convert(
+			TiltakDeltaker.Status.JATAKK,
+			now,
+			tomorrow,
+			null,
+			null,
+			gjennomforingStatus
+		).navn shouldBe VENTER_PA_OPPSTART
+	}
+	"status - JATAKK og har sluttdato i fortid - returnerer HAR_SLUTTET" {
```

Comment:
Er litt skeptisk til denne altså.. tror ikke de har gått på tiltaket om vi laster inn gamle data med JATAKK status..

Reply:
Dette er jo et case som i følge Lars ikke skal skje (men siden Arena er Arena bør vi ta høyde for det uansett). Hvis Arena publiserer gamle hendelser på nytt, eller hvis vi leser topicen på nytt, vil det vel kunne skje at vi mottar en statusmelding med statusen JATAKK der sluttdato er tilbake i tid. Men da vil det vel komme en ny melding som korrigerer statusen etterpå uansett? 

Vi burde forresten kanskje gjøre sjekken på om datoer er passert mot tidspunktet meldingen ble sendt fra arena (hvis det finnes) og ikke sammenligne med i dag, for hvis vi leser at topicen fra start igjen blir det jo feil statuser..?

Reply:
Nå skjønte jeg at det var vel dette med lasting av gamle data som du egentlig mente 😅 Og ja, det tenker jeg også blir rart. Hvis vi skal få det samme resultatet hver gang vi leser den samme meldingen så kan vi ikke bruke i dag for å avgjøre riktig status, da må vi bruke et timestamp fra meldingen. Jeg ser at meldingen inneholder blant annet REG_DATO, MOD_DATO og DATO_STATUSENDRING, kanskje vi kan bruke en av disse for å sjekke om startdato/sluttdato er passert? Da tenker jeg i så fall på for alle statusene siden det vil kunne bli feil hvis vi leser gamle data for alle som sjekker noe i meldingen mot dagens dato.